docs/source/conf.py

```diff
@@ -34,7 +35,12 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.napoleon']
+# isort: unique-list
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.viewcode',
```

Comment:
The width on the viewcode stuff seems to assume 80 columns.  Probably should follow up with being able to widen that (it might be viewcode_tab_width, not certain).

Reply:
correction, seems to maybe need a custom css. it can wait